Десятая лабораторная работа выполнена

PopovMatvey opened the pull request:
1. Были выполнены задания 1-2;
2. Было выполнено задание для Library (свёрстан header и footer).

PopovMatvey committed:
Create task-01.css

PopovMatvey committed:
Add files via upload

PopovMatvey committed:
Десятая лабораторная работа выполнена (tasks)

Были выполнены задания 1-2.

PopovMatvey committed:
Create style.css

PopovMatvey committed:
Add files via upload

PopovMatvey committed:
Create avatar-01.png

PopovMatvey committed:
Add files via upload

PopovMatvey committed:
Десятая лабораторная работа выполнена (Library)

Был свёрстан header и footer страницы.

djilua1 commented:
Библиотеку отдельным ПРом нужно делать. Разбей на 2. В этом буду проверять только то, что касается первых двух заданий.

djilua1 commented on src/ex10_html-css-advanced/assets/css/task-01.css:
```diff
@@ -0,0 +1,148 @@
+    100% {
+        display: none;
+        opacity: 0;
+        transform: rotate(1080deg);
```
Сделай бесконечную загрузку.

djilua1 commented on src/ex10_html-css-advanced/task-02.html:
```diff
@@ -0,0 +1,44 @@
+    </main>
+</body>
+
+</html>
```
Анимация должна появляться только при наборе символов.

djilua1 commented on src/ex10_html-css-advanced/assets/css/task-02.css:
```diff
@@ -0,0 +1,84 @@
+
+.field-specific-location {
+    width: 70%;
+    padding: 0px 0px 0px 20px;
```
padding-left?

djilua1 commented on src/ex10_html-css-advanced/assets/css/task-02.css:
```diff
@@ -0,0 +1,84 @@
+
+.field-description {
+    width: 100%;
+    padding: 0px 0px 30px 0px;
```
pfdding-bottom?

djilua1 commented on src/ex10_html-css-advanced/assets/css/task-02.css:
```diff
@@ -0,0 +1,84 @@
+    width: 100px;
+    height: 30px;
+    margin: 0;
+    padding: 0;
```
Для кнопок можно в общие вынести.

djilua1 commented on src/ex10_html-css-advanced/assets/css/task-02.css:
```diff
@@ -0,0 +1,84 @@
+    padding: 20px;
+    margin-top: 20px;
+    font-size: 20px;
+    border-bottom: 1px solid #bdbdbd;
```
Старайся писать стили цветом в одном формате. Я бы посоветовал через #.

djilua1 commented:
Также отдельно прикрепи в систему отдельно библиотеку. И фич ветку отдельную сделай для нее, чтобы лить не в мастер, а в нее. В конце создашь большой ПР со всеми файлами в мастер.

PopovMatvey committed:
Delete index.html

PopovMatvey committed:
Delete src/library_task/assets directory

PopovMatvey committed:
Add files via upload

PopovMatvey committed:
Исправления по десятой лабораторной работы

Были исправлены все шесть замечаний:
1. Для загрузчика была сделана бесконечная загрузка;
2. Теперь анимация срабатывает только при наборе символов;
3 и 4. Общую запись padding изменил на padding-left и padding-bottom;
5. Стили для кнопок были вынесены в общие;
6. Все стили для цвета теперь одном формате ( через # ).
